packages/keystone/schema.js

```diff
@@ -142,20 +142,6 @@ async function find (schemaName, condition) {
     return await schemaList._keystone.lists[schemaName].adapter.find(condition)
 }
 
-/**
- * @param {string} schemaName
- * @param {Object} condition
- * @returns {Promise<number>}
- */
-async function count (schemaName, condition) {
```

Comment:
why did you remove count?

Reply:
Seems not relative to this PR

Reply:
I was asked about deleting of this unused function by @pahaz and decided to do it within this PR.